webpack.test.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  devtool: 'inline-source-map',
+
+  target: 'node',
+
+  ts: {
+    configFileName: 'tsconfig.tests.json',
+  },
+
+  resolve: {
+    extensions: ['', '.ts', '.js'],
+  },
+
+  module: {
+    loaders: [
+      {
+        test: /\.ts$/,
+        loaders: ['ts'],
+      },
+    ],
+  },
+};
+
+const helpers = {};
+const path = require('path');
+
+const rootPath = path.resolve(__dirname, '..');
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).